src/_pytest/pytester.py

```diff
@@ -696,15 +688,14 @@ def __init__(
         #: be added to the list.  The type of items to add to the list depends on
         #: the method using them so refer to them for details.
         self.plugins: List[Union[str, _PluggyPlugin]] = []
-        self._cwd_snapshot = CwdSnapshot()
         self._sys_path_snapshot = SysPathsSnapshot()
```

Comment:
should we log a followup to ensure sys.path/sys.modules handling in coordination with monkeypatch

Reply:
Might be possible but some complications:

- `SysPathSnapshot` also saves `sys.meta_path`, monkeypatch doesn't.
- Monkeypatch doesn't save `sys.path` by default, though pytester can probably force it to do it
- `SysModulesSnapshot` has no equivalent in monkeypatch currenty as far as know. Also has some `preserve_modules` exclusion...